<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kalendarz</class>
 <widget class="QDialog" name="kalendarz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>261</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>261</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Organizator</string>
  </property>
  <widget class="QCalendarWidget" name="cw_kalendarz">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pb_dodaj">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb_edytuj">
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pb_usun">
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pb_pokaz">
      <property name="text">
       <string>Pokaż</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="lw_spotkania">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>181</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
